src/features/auth/register-form/RegisterForm.tsx

```diff
@@ -2,79 +2,113 @@ import { FunctionComponent } from "react";
 import TextField from "@mui/material/TextField";
 import Button from "@mui/material/Button";
 import useRegisterFormState from "./use-register-form-state";
+import useTheme from "@mui/material/styles/useTheme";
+import useMediaQuery from '@mui/material/useMediaQuery';
+import Box from "@mui/material/Box";
+import Typography from '@mui/material/Typography';
+import ucabLogo from "../images/ucabLogo.webp";
+import Link from '@mui/material/Link';
+import {TextFieldTypography,
+    ImageBoxStyles, 
+    TextFieldStyles, 
+    ButtonStyles, 
+    UnderFormStyles, 
+    UcabLogoStyles} 
+from "./styles/RegisterFormStyles.tsx";
 
-const RegisterForm: FunctionComponent<props> = ({ disabled, onSubmit }) => {
-  const {
-    name,
-    lastname,
-    email,
-    password,
-    confirmPassword,
-    onChangeName,
-    onChangeLastname,
-    onChangeEmail,
-    onChangePassword,
-    onChangeConfirmPassword,
-  } = useRegisterFormState();
-  return (
-    <>
-      <TextField
-        value={name}
-        label="Nombre"
-        onChange={onChangeName}
-        disabled={disabled}
-      />
-      <TextField
-        value={lastname}
-        label="Apellido"
-        onChange={onChangeLastname}
-        disabled={disabled}
-      />
-      <TextField
-        value={email}
-        label="Correo electronico"
-        type="email"
-        onChange={onChangeEmail}
-        disabled={disabled}
-      />
-      <TextField
-        value={password}
-        label="Contraseña"
-        type="password"
-        onChange={onChangePassword}
-        disabled={disabled}
-      />
-      <TextField
-        value={confirmPassword}
-        label="Confirmar contraseña"
-        type="password"
-        onChange={onChangeConfirmPassword}
-        disabled={disabled}
-      />
-      <Button
-        variant="contained"
-        color="primary"
-        type="submit"
-        disabled={disabled}
-        onClick={() => {
-          onSubmit(name, lastname, email, password, confirmPassword);
-        }}
-      >
-        Registrarse
-      </Button>
+const RegisterForm: FunctionComponent<props> = ({disabled, onSubmit}) =>{
+    const {name, email, password, confirmPassword, onChangeName, onChangeEmail, onChangePassword, onChangeConfirmPassword} = useRegisterFormState();
+    
+
+    const theme = useTheme();
+    const isSmallScreen = useMediaQuery(theme.breakpoints.down('sm'));
+    const FormBoxStyles = {
+        width: isSmallScreen ? '85%' : '350px',
```

Comment:
```suggestion
        width: isSmallScreen ? '85%' : '350px',
```

Es una mala practica usar %  para este tipo de cosas, para ambos tamaños usa un witdh fijo